$MirOS: ports/devel/libreadline/patches/patch-support_shobj-conf,v 1.3 2008/04/22 17:24:18 bsiegert Exp $
--- support/shobj-conf.orig	Tue Aug 26 13:44:31 2008
+++ support/shobj-conf	Tue Aug 26 13:44:51 2008
@@ -182,7 +182,7 @@ darwin*|macosx*)
 	SHLIB_LIBS='-lncurses'	# see if -lcurses works on MacOS X 10.1 
 	;;
 
-openbsd*|netbsd*)
+openbsd*|netbsd*|mirbsd*)
 	SHOBJ_CFLAGS=-fPIC
 	SHOBJ_LD='${CC}'
 	SHOBJ_LDFLAGS='-shared'
